.github/workflows/pull-request.yml

```diff
@@ -73,14 +73,19 @@ jobs:
         with:
           time: '30s'
 
+      - name: Grant permission for db data
```

Comment:
@riyenas0925 I updated the workflow to **grant permission** for db data. I hope this change can solve the problem.

Reply:
LGTM 👍

Reply:
Shouldn't we add permission change to `db/data` in` release-publish.yml` as well?

Reply:
The `db/data` is created by running `docker:db`. There was no related step in the `release-publish.yml`, so I excluded it.

Reply:
@jongwooo , I mean, even in release-publish, there will be an error saying that it does not have db/data permission when building, so we need to add the db/data permission change to release-publish.yml as well.

Reply:
Oh, it's that release doesn't run docker. now i understand